### PR TITLE
Fixes #45: Mention ADR-004 in index.md

### LINKED ISSUE #45: Write an ADR about why we do open-source development
Name: dev-rel-docs/adr/004-opensource-development.md

https://dev.azure.com/CFIA-DevOps-ACIA/AI-Lab/_wiki/wikis/AI-Lab.wiki/757/Why-OpenSource-development-at-the-AI-Lab

### PR DESCRIPTION
**Pull request by RussellJimmies**
#45

### CODE REVIEW

**Comment by rngadam** [comment]
Have you tested this? If I do view file and click on it I get:

image

...and clearly, the path adr/adr is wrong

**Review by rngadam — CHANGES_REQUESTED** [review]
Have you tested this? If I do view file and click on it I get:

image

...and clearly, the path adr/adr is wrong

**Comment by RussellJimmies** [comment]
No I didn't test, I simply followed how it was done initially. This should work now.

File: adr/index.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Index of all ADRs
 
-* [ADR-001: AI-Lab using ADR](./adr/001-ailab-using-adr.md) - Why we decided to
+* [ADR-001: AI-Lab using ADR](./001-ailab-using-adr.md) - Why we decided to
   use Architectural Decision Records.
 
-* [ADR-001 - Attachment: ADR Template](./adr/001-ailab-using-adr-template.md) -
+* [ADR-001 - Attachment: ADR Template](./001-ailab-using-adr-template.md) -
```
Comment (rngadam) [inline adr/index.md:6]: do you need ```./``` on your files? by default I think it should be the current directory.

**Review by rngadam — CHANGES_REQUESTED** [review]

File: adr/index.md (hunk 1 above)
Comment (RussellJimmies) [reply adr/index.md]: Tested it, didn't work without `./`
Fixed adr-001 links.

**Comment by RussellJimmies** [comment]
Closed in #78

**Pull request closed without merging**